inc/functions/admin.php

```diff
@@ -75,32 +78,17 @@ function get_imagify_admin_url( $action = 'options-general', $arg = array() ) {
  * @return array An array containing the max width and height.
  */
 function get_imagify_max_intermediate_image_size() {
-	global $_wp_additional_image_sizes;
-
 	$width  = 0;
 	$height = 0;
 	$limit  = 9999;
-	$sizes  = array( 'thumbnail' => 1, 'medium' => 1, 'large' => 1 );
-	$get_intermediate_image_sizes = get_intermediate_image_sizes();
-
-	// Create the full array with sizes and crop info.
-	foreach ( $get_intermediate_image_sizes as $_size ) {
-		if ( isset( $sizes[ $_size ] ) ) {
-			$_size_width  = get_option( $_size . '_size_w' );
-			$_size_height = get_option( $_size . '_size_h' );
-		} elseif ( isset( $_wp_additional_image_sizes[ $_size ] ) ) {
-			$_size_width  = $_wp_additional_image_sizes[ $_size ]['width'];
-			$_size_height = $_wp_additional_image_sizes[ $_size ]['height'];
-		} else {
-			continue;
-		}
 
-		if ( $_size_width < $limit ) {
-			$width = max( $width, $_size_width );
+	foreach ( get_imagify_thumbnail_sizes() as $_size ) {
```

Comment:
What do you mean? The function `get_imagify_thumbnail_sizes()` is used in some other places in the plugin.
Do you say that because it's in a foreach?